docs/application/native/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,634 @@
+            <label xml:lang="ko-kr">base-frame[KOR]</label>
+        </frame-component>
+    </component-based-application>
+    <privileges>
+        <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+    </privileges>
+</manifest>
+```
+
+Following are the main attributes:
+
+- `id`
+
+  Every component element has an `id` attribute. The attribute ID must be globally unique in a device. If not, the application installation fails. This attribute is also used by other applications to confirm which component must be created.
+
+- `main`
+
+  Component-based applications can have multiple components and there is one main component. The main component is the component that is created when the component-based application receives a launch request without a specific component ID. To set the main component, set the `main` attribute to `true`.
+
+- `icon-display`
+
+  This attribute is only for the frame-component element. If this value is `true`, the home application will display an icon on the app tray for this frame-component.
+
+- `launch_mode`
+
+  This attribute is only for the frame-component element. Application launch mode (available values: `single` (launched as a main application), `group` (launched as a sub application), `caller` (caller application [defines the launch mode](../app-management/app-controls.md#mode) with the `app_control_set_launch_mode()` function))
+
+- `taskmanage`
+
+  This attribute is only for the frame-component element. Indicates whether the application is shown in the task manager (available values: `true`, `false`)
+  By default, this value is set to `single`.
+
+## Prerequisites
+
+To use the functions and data types of the component-based application API (in [mobile](../../api/mobile/latest/group__CAPI__APPLICATION__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__APPLICATION__MODULE.html) applications), include the `<component_based_app.h>` header file in your application:
+
+```
+#include <component_based_app.h>
+```
+
+<a name="application_monitoring"></a>
+## Running Component-based Application
+
+To run a component-based application, you must register the component-based application callbacks and start a main event loop as following codes:
+
+```
+component_class_h __app_create_cb(void *user_data)
+{
+    component_class_h comp_class = NULL;
+    return comp_class;
+}
+
+void __app_terminate_cb(void *user_data)
+{
+}
+
+int main(int argc, char** argv)
+{
+    int ret;
+    component_based_app_lifecycle_callback_s callback = {
+        .create = __app_create_cb,
+        .terminate = __app_terminate_cb
+    };
+
+    ret = component_based_app_main(argc, argv, &callback, NULL);
+    if (ret != APP_ERROR_NONE)
+        return ret;
+
+    return 0;
+}
+```
+
+<a name="frame_component_monitoring"></a>
+## Managing Frame-component
+
+To add and manage frame-components, you must add frame-component and register the frame-component callbacks.
```

Comment:
Please add "a" before "frame-component". 
... you must add a frame-component and register the frame-component callbacks.